llvm/test/CodeGen/AArch64/aarch64-neon-bici.ll

```diff
@@ -0,0 +1,125 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -mtriple=aarch64-neon < %s | FileCheck %s
+
+declare <8 x i16> @llvm.aarch64.neon.uhadd.v8i16(<8 x i16>, <8 x i16>)
+declare <8 x i16> @llvm.aarch64.neon.urhadd.v8i16(<8 x i16>, <8 x i16>)
+declare <8 x i16> @llvm.aarch64.neon.shadd.v8i16(<8 x i16>, <8 x i16>)
+declare <8 x i16> @llvm.aarch64.neon.srhadd.v8i16(<8 x i16>, <8 x i16>)
```

Comment:
These need pre-committing to trunk so the patch shows the codegen diff

Reply:
Should I make a separate PR for this test?

Reply:
Yeah that sounds good, if you don't have access to push them directly. Hopefully it is a simple one to get committed either way.